download.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+set -e
+
+test -z "$VERSION" && {
+	echo "Please specify the version by setting the VERSION env var." >&2
+	exit 1
+}
+test -z "$FLAVOUR" && {
```

Comment:
Two very small naming suggestions, but I don't feel too strongly about them:
1. Renaming `flavour` to `distribution`
2. Renaming folder `otel-dotnet-auto` to `otel-dotnet-auto-installation`

Reply:
I like first suggestion 😉